monkestation/code/modules/clothing/suits/coats.dm

```diff
@@ -104,6 +104,13 @@
 	icon_state = "guardman_vest"
 
 
+/obj/item/clothing/suit/armor/blueshield
+	name = "blueshield coat"
+	desc = "A stylish kevlar coat worn by the blueshield agent."
+	icon = 'monkestation/icons/obj/clothing/suits.dmi'
+	worn_icon = 'monkestation/icons/mob/clothing/suit.dmi'
+	icon_state = "blueshield"
+
```

Comment:
Doesn't look like you have an armor value? Don't know if armor inherits a basic one but you may want to make your own armor values.